tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/GenericAuthorityTests.cs

```diff
@@ -21,28 +22,37 @@ namespace Microsoft.Identity.Test.Unit.CoreTests.InstanceTests
     public class GenericAuthorityTests : TestBase
     {
         [DataTestMethod]
-        [DataRow(true)]
-        [DataRow(false)]
+        [DataRow(true, false)]
```

Comment:
Consider creating new tests. Adding more and more options to existing tests makes them hard to read.